Tasks/AzureWebPowerShellDeployment/Publish-AzureWebDeployment.ps1

```diff
@@ -51,18 +51,18 @@ $packageFile = Get-SingleFile $packageFile $Package
 #If we're provided a WebSiteLocation, check for it and create it if necessary
 if($WebSiteLocation)
 {
-    Write-Host "Get-AzureWebSite -Name $WebSiteName -ErrorAction SilentlyContinue"
-    $azureWebSite = Get-AzureWebSite -Name $WebSiteName -ErrorAction SilentlyContinue
+    Write-Host "Get-AzureWebSite -Name `"$WebSiteName`" -ErrorAction SilentlyContinue"
+    $azureWebSite = Get-AzureWebSite -Name `"$WebSiteName`" -ErrorAction SilentlyContinue
     if(!$azureWebSite)
     {
-        Write-Host "New-AzureWebSite -Name $WebSiteName -Location $WebSiteLocation"
-        $azureWebSite = New-AzureWebSite -Name $WebSiteName -Location $WebSiteLocation
+        Write-Host "New-AzureWebSite -Name `"$WebSiteName`" -Location $WebSiteLocation"
+        $azureWebSite = New-AzureWebSite -Name `"$WebSiteName`" -Location $WebSiteLocation
     }
 }
 
 #Deploy the package
 $azureCommand = "Publish-AzureWebsiteProject"
-$azureCommandArguments = "-Name $WebSiteName -Package `"$packageFile`" $AdditionalArguments"
+$azureCommandArguments = "-Name `"$WebSiteName`" -Package `"$packageFile`" $AdditionalArguments"
 $finalCommand = "$azureCommand $azureCommandArguments"
 Write-Host "finalCommand= $finalCommand"
```

Comment:
Can we change this to just Write-Host "$finalCommand"  I don't care about the internal variable name as a user